docs/utxo-hd/v0.1/for-public.md

```diff
@@ -0,0 +1,166 @@
+# UTxO-HD for the general public
+
+This document aims to provide a comprehensible explanation of what UTxO-HD is
+and what it means for the end users of the Cardano blockchain.
+
+As UTxO-HD is mostly an internal improvement, there will not be many technical
+details in this document, instead feel invited to read the [documentation for
+developers](../for-developers.md) for more information.
+
+# Why UTxO-HD
+
+Over time, the `cardano-node` implementation's memory usage has been steadily
+increasing. A sync with mainnet shows that currently the node uses around 16GBs
+of memory at its peak. Not every user of the Cardano blockchain has a computer
+with that much memory and therefore it becomes necessary to reduce the memory of
+the running node.
+
+This poses two main goals for this project:
+
+1. In order to promote decentralization of the network, more users should be
+   able to participate actively. Nodes running on a Raspberry Pi or shared
+   instances should be capable of keeping up with the network. Disk storage is
+   cheaper than memory and as long as the overall performance of the network is
+   not impacted, these machines should be perfectly acceptable to run a node.
+
+2. The memory consumption of a node will be in principle forever increasing as
+   more data gets into the blockchain. The UTxO set can grow and so can other
+   data structures such as delegations or rewards. UTxO-HD opens the door for
+   moving part of this data to the disk, which in the long run will aid with the
+   sustainability of Cardano.
+
+UTxO-HD is the first step of this solution, which moves the UTxO set into a
+disk-backed storage. This effectively lowers the memory by 5GB (tentative
+numbers shown by ad-hoc benchmarks, should not be considered exact). However
+this comes at a cost, both in speed performance and in complexity of the code.
+
+We have implemented two different backends for this feature:
+
+- `InMemory`: enabled by default. Instead of using the disk, it uses an
+  in-memory storage to keep the UTxO set. It is roughly as fast as the
+  pre-UTxO-HD node, but it doesn't contribute to lowering the memory, as the
+  data retained is roughly the same. Intended to be used by users with powerful
+  machines.
+- `LMDB`: enabled by the `--lmdb-ledger-db-backend` flag for the node. Uses a
+  [LMDB](http://www.lmdb.tech/doc/) backend on the disk to lower the memory
+  usage of the node. It is slower as it has to perform I/O to get the data, but
+  it successfully reduces the memory consumption of the node.
+
+# Impact of the solution
+
+We have measured the performance of the node after implementing UTxO-HD and we
+have observed 4 different degrees of impact depending on the operation
```

Comment:
```suggestion
have observed 4 different types of impact depending on the operation
```

Reply:
Or "below we describe the impact in 4 different scenarios".

Reply:
Done